markdown_lab_rs.py

```
@@ -41,19 +53,175 @@
+
+    Returns:
+        Converted content in the specified format
+    """
     if RUST_AVAILABLE:
         try:
-            return _rs_convert_html_to_markdown(html, base_url)
+            return _rs_convert_html_to_format(html, base_url, output_format.value)
         except Exception as e:
             logger.warning(
-                f"Error in Rust HTML-to-markdown conversion, falling back to Python: {e}"
+                f"Error in Rust HTML conversion to {output_format}, falling back to Python: {e}"
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)